### PR TITLE
Feature develop merge tf2.4

### PR DESCRIPTION
**Pull request by aniktash**

### CODE REVIEW

**Comment by aniktash** [comment]
Intended to be TF 2.4, but downgraded to 2.3 for now to merge with latest develop.

File: requirements-cu111.txt [hunk 1]
```diff
@@ -9,8 +9,8 @@ Pillow>=7
 shap>=0.34.0
 tk>=0.1.0
 sndfile>=0.2
-https://download.pytorch.org/whl/cu110/torch-1.7.0%2Bcu110-cp38-cp38-linux_x86_64.whl
-https://download.pytorch.org/whl/cu110/torchvision-0.8.1%2Bcu110-cp38-cp38-linux_x86_64.whl
+https://download.pytorch.org/whl/cu110/torch-1.7.1%2Bcu110-cp38-cp38-linux_x86_64.whl
+https://download.pytorch.org/whl/cu110/torchvision-0.8.2%2Bcu110-cp38-cp38-linux_x86_64.whl
 tensorboard==2.3.0
 numba<0.50.0
 opencv-python>=4.4.0
```
Comment (vicloginov) [inline requirements-cu111.txt:16]: Add pytsmod>=0.3.3

**Review by vicloginov — COMMENTED** [review]
Add pytsmod>=0.3.3 to requirements-cu111.txt

**Review by vicloginov — APPROVED** [review]
Looks good

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
There's a missing line break in requirements.txt
"tensorboard==2.4.0numba<0.50.0"

**Pull request merged**